Automattic/jetpack
# Sync: Fix how we enqueue term_relationships on full sync 🏝

Fixes how we iterate over al `term_relationships` items.

#### Testing instructions:
* Find a site with a lot of `term_relationships` 
* Disable sync on your site. (you can use the debugger)
* Add some `term_relationships`
* Start a full sync that includes `term_relationships` (from wpcom: `wp jetpack-sync start --blog-id=144249210 --object-type=term_relationships`)
* Make sure the new `term_relationships` get sync.
* Test `test_full_sync_sends_all_term_relationships` should fail on `master` and pass on this branch
